### ISSUE TITLE
How can i connect 2 Controllers?

### ISSUE THREAD

**Issue by Splitty1337** [Jays2Kings/DS4Windows#475]
Hello,
(for the first,i want to say sorry for some language-mistakes)
i wanted to play Monopoly Plus with friends on my computer,but we want to play with PS4 Controllers,i connected 1 Controller and that worked perfect,but i want that every player have their own controller and connected a second controller,the first controller worked still,but the second did not worked,the touchpad worked,but not the buttons. I read that i should activate "hide ds4 controller" but with that setting my controllers wasnt anymore connected and i had a lot of problems to fix that . 
I hope you can help me.

**Comment by iemtheejay** [comment]
Yup im having the same challenges as well. Im not sure how to connect atleast 2 controllers at the same time using ds4 windows. Please let us know or if there a video tutorial that we can watch. Thank you!
